packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/baseTemplateCommand.test.ts

```diff
@@ -61,8 +61,8 @@ describe('FilePathExistsChecker for Aura bundle', () => {
     });
 
     after(() => {
-      sinon.restore(vscode.workspace);
-      sinon.restore(notificationService);
+      findFilesSpy.restore();
```

Comment:
Fixes some flapping tests caused by new proposed api's in version 1.36 of vscode.